### ISSUE TITLE
[Bug]: Cannot use Matchmaker w/o Server

### ISSUE THREAD

**Issue by hunkydoryrepair** [colyseus/colyseus#695]
### Context

Up until @colyseus/core 0.15.17, we were successfully using Matchmaker on some instances without creating the Colyseus server. This was helpful as about 70% of our CPU load is handling endpoints, and it allowed us to run Colyseus Monitor and some other endpoints for monitoring server and finding rooms w/o ever running any rooms. The only thing we couldn't do was call createRoom.

### Bug description

As of 0.15.20, this breaks because `stats.reset()` is called in the matchMaker.setup() function.  We can call `stats.excludeProcess` immediately after, but that is possibly too late to avoid errors. We get "room not defined" when another server tries to connect and create a room as soon as our new "server" shows up with 0 rooms. 

Basically, the server was never registered, but it shows up in the stats. I wonder if there could be an option to not initialize the stats from matchmaker?

### Steps to reproduce

1. On server A, initialize Colyseus normally
2. On server B, initialize just matchmaker so we can monitor and use matchmaker functionality to access redis information about server A

### Environment & Versions

Colyseus version: 0.15.20

**Comment by endel** [comment]
HI @hunkydoryrepair, thank you for the feedback, I didn't consider this scenario - will prepare a fix for this today 🙏
